src/editor/components/colorSelector.js

```diff
@@ -0,0 +1,127 @@
+import { mdiClose } from '@mdi/js';
+import { fireEvent } from 'custom-card-helpers';
+import { css, html, LitElement } from 'lit-element';
+
+export const colorSelector = {
+  hex_color: {},
+};
+
+export class CustomColorSelector extends LitElement {
```

Comment:
Is there a reason not to use the color picker provided by hass itself?

If it is the "clearable" feature, it could be handled in the usual hass way by having an enable/disable toggle in front of the color picker.

If it is the hex format, could it be converted "output-only" in the main `valueChanged` function? I am not familiar enough with the way these things work...

Reply:
Mainly it was the hex format. I've tried to do the convertion on output but this resulted in warnings in console about supplied color value to hass color picker but the selecter was rendering just fine.

Reply:
Ah, I see... I'd guess, we'd have to convert back and forth to not loose colours on each startup of the editor... At least in `setConfig`, `configChanged` *and* `entitiesChanged`... Then use `[r,g,b]` internally in the editor and hex outside... 

Another idea: How about we allow `[r,g,b]` as a new colour input format for the mini-graph-card itself? Better suited for a follow-up PR then, maybe... We'd still loose non-hex colours on first startup of the editor, I assume... 

I shall test, if losing colours is actually an issue...

Reply:
I did a test, and yes, loosing colours is an issue. Try this: 
* Edit card
* Switch to YAML
* Set colour to "yellow" on an entity
* Switch to UI mode and edit entity
* Observe the colour to be black now.

Reply:
I've addressed this issue in the latest commit